packages/smooth_app/lib/pages/preferences/country_selector/openfoodfacts_country_iso2code_extension.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:openfoodfacts/openfoodfacts.dart';
+
+extension OpenFoodFactsCountryIso2Extension on OpenFoodFactsCountry {
+  String get iso2Code =>
+      offTag.toUpperCase() == 'UK' ? 'GB' : offTag.toUpperCase();
```

Comment:
```suggestion
  String get iso2Code =>
      this == OpenFoodFactsCountry.UNITED_KINGDOM ? 'GB' : offTag.toUpperCase();
```
Generally speaking it's not a brillant idea to copy code.